File: Frontend/angular-app/src/app/modules/inventory/entities/Assets.ts
export class Assets {
  public id: number | undefined;
  public warehouseId: number | undefined;
  public commodityId: number | undefined;
  public measureId: number | undefined;
  public assetGroupId: number | undefined;
  public unitId: number | undefined;
  public documentDate: string | undefined;
  public commoditySerial: string | undefined;
  public assetSerial: string | undefined;
  public depreciationTypeBaseId: number | undefined;
  public price: number | undefined;
  public depreciatedPrice: number | undefined;
  public isActive: boolean | undefined;
  public documentItemId: number | undefined;
  public startWithNumber: number | undefined;
  public prefix: string | undefined;
  public warehousesTitle: string | undefined;
  public categoryTitle: string | undefined;
  public measureTitle: string | undefined;
  public searchTerm: string | undefined;
  public categoryLevelCode: string | undefined;
  public commodityTitle: string | undefined;
  public unitsTitle: string | undefined;
  public depreciationTitle: string | undefined;
  public assetGroupTitle: string | undefined;
  public commodityCode: string | undefined;
  public isHaveWast: boolean | undefined;
  public isAsset: boolean | undefined;
  public isConsumable: boolean | undefined;
  public documentNo: number | undefined;
  public description: string | undefined;
  public title: string | undefined;

  public documentItemsDescription: string | undefined;
  public documentDescription: string | undefined;
  public totalDescription: string | undefined;
  public assetsSerials: AssetsSerial[] | undefined;
}
export class AssetsSerial {
  public commodityId: number | undefined;
  public serial: string | undefined;
  public id: number | undefined;
  public selected: boolean | undefined;
  public title: boolean | undefined;
  public commoditySerial: boolean | undefined;
  public description: string | undefined ;
  

}

File: Backend/Microservices/Inventory/Eefa.Inventory.Domain/Aggregates/Assets/Assets.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using Eefa.Common.Data;
using Eefa.Common.Domain;

namespace Eefa.Inventory.Domain
{

    [Table("Assets", Schema = "inventory")]
    public partial class Assets : DomainBaseEntity, IAggregateRoot
    {
        public int WarehouseId { get; set; } = default!;
        public int CommodityId { get; set; } = default!;
        public int MeasureId { get; set; } = default!;
        public int AssetGroupId { get; set; } = default!;
        public int? UnitId { get; set; } = default!;
        public DateTime DocumentDate { get; set; } = default!;
        public string CommoditySerial { get; set; } = default!;
        public string AssetSerial { get; set; } = default!;
        public int? DepreciationTypeBaseId { get; set; } = default!;
        public double Price { get; set; } = default!;
        public double? DepreciatedPrice { get; set; } = default!;
        public bool IsActive { get; set; } = default!;
        public int? DocumentHeadId { get; set; } = default!;
        public int? DocumentItemId { get; set; } = default!;
        public string Description { get; set; } = default!;
       
    }
   

}
